source/frontend/src/features/mapSideBar/compensation/list/CompensationListView.tsx

```diff
@@ -119,7 +100,7 @@ export const CompensationListView: React.FunctionComponent<ICompensationListView
             title="Compensation Requisitions"
             addButtonText="Add Requisition"
             addButtonIcon={<FaPlus size={'2rem'} />}
-            onAdd={onAdd}
+            onAdd={statusUpdateSolver?.canEditOrDeleteCompensation() ? onAdd : null}
```

Comment:
on other components we show a tooltip that they cannot "Add" but I don't see it here. Just confirming this is expected or whether the tooltip should be displayed here...